script/script.js

```diff
@@ -0,0 +1,85 @@
+const ham = document.querySelector('.ham');
+const navbar = document.querySelector('.navbar-hide');
+
+ham.addEventListener('click', () => {
+  navbar.classList.add('navbar');
+  navbar.classList.toggle('navbar-hide');
+});
+
+function openCloseMenu() {
+  ham.classList.toggle('change');
+}
+
+ham.addEventListener('click', openCloseMenu);
+
+const speakers = [
+  {
+    image: '../instructors/b1.jpg',
```

Comment:
- [ ] **Required:** Kindly tweak your `dir`. Your images are not displayed on the live page. It should be `image: './instructors/b1.jpg'` rather than the current `image: '../instructors/b1.jpg'`